projects/planner/src/app/plan/diveoptions/diveoptions.component.html

```diff
@@ -160,6 +160,15 @@
                                 Needs to be number {{ranges.lastStopDepthLabel}}
                             </div>
                         </div>
+                        <div class="mb-4 col col-12 col-sm-6 col-md-3 col-xxl-6">
+                            <label for="decoStopDistance" class="form-label" mdbLabel>Distance between deco stops [{{units.length}}]:</label>
+                            <input id="decoStopDistance" formControlName="decoStopDistance" class="form-control" type="number"
+                                (input)="applyOptions()" required min="1" max="10" step="1"
```

Comment:
The min. and max values need to come from units.ranges see UnitConversions.ts,
they cant be fixed values, since they depend on the current unit system.